src/Inquirer/Validators/RangeValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Win32.SafeHandles;
+using System;
+
+namespace InquirerCore.Validators
+{
+    public class RangeValidator : IValidator
+    {
+    private int iMin { set; get; }
+    private int iMax { set; get; }
```

Comment:
In C# we don't use Hungarian notation, you could just use Min and Max.
Also after open a bracket we always indent one more level.

Reply:
About getters and setters, which declaration is prefered (https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/get)?

Reply:
I think that the getters and setters are OK the way you made 😃

Reply:
ok